src/App.js

```diff
@@ -49,8 +53,10 @@ class App extends Component {
                height={item.height}     // Height - used for a CSS style to position on the screen
                layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
                key={item.id}            // Key - to help React with performance
-
+               type={item.type}
+               scored={this.onItemClicked}
```

Comment:
I would give this a more helpful name in the props, because on it's own it doesn't tell us what's happening.